Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs

```diff
@@ -1335,5 +1365,25 @@ protected override void ProcessRecord()
                 instances = request.Execute().ManagedInstances;
             }
         }
+
+        private void WaitRegionManagedInstance(string project, string region, string name)
```

Comment:
Suggestion: Create a common function for this and the above that takes a ClientServiceRequest and a string message to print while waiting.  The bulk of the logic is shared.